docs/product-user-guide/src/main/asciidoc/dmn-elements-ref.adoc

```diff
@@ -0,0 +1,10 @@
+[id='dmn-elements-ref']
+= DMN Elements
+
+DMN models consist of the following four elements:
+
```

Comment:
This section only makes sense if the explanation refers to the elements of the diagram. Can we add an image of a diagram and refer the explanation to each node?